packages/react/src/components/List/List.mdx

```diff
@@ -45,7 +45,7 @@ const yankees = {
 ## Props
 
 | Name                       | Type                                                                                                                                                 | Default                                         | Description                                                                                                                                                                                                                                                                                                                   |
-| :------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| :------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- | --- | ------------------------------------------------------------------ |
```

Comment:
Seems like two additional columns were accidentally added here. You can view/preview your documentation changes by clicking on the docs tab for any List story.

![image](https://user-images.githubusercontent.com/5167091/149503545-91f7369b-d257-4b52-adf5-3baaf0bb4759.png)